File: abc/abc214/a.java
// https://atcoder.jp/contests/abc214/submissions/41240852

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    if(n <= 125) System.out.print(4);
    else if(n <= 211) System.out.print(6);
    else System.out.print(8);
  }
}

File: abc/abc214/a.cs
// https://atcoder.jp/contests/abc214/submissions/46467156

using System;

class A
{
  static void Main()
  {
    int n = int.Parse(Console.ReadLine());

    if(n <= 125)
    {
      Console.WriteLine(4);
    }
    else if(n <= 211)
    {
      Console.WriteLine(6);
    }
    else
    {
      Console.WriteLine(8);
    }
  }
}

